src/Symfony/Component/Serializer/Exception/InvariantViolationException.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Exception;
+
+use Symfony\Component\Serializer\InvariantViolation;
+
+final class InvariantViolationException extends \RuntimeException implements ExceptionInterface
+{
+    /**
+     * @var array<string, array<InvariantViolation>>
+     */
```

Comment:
phpdoc isn't necessary here because it's on the constructor

Reply:
Removed.